tests/main/auto-refresh-pre-download/task.yaml

```diff
@@ -50,6 +50,7 @@ execute: |
 
     if [ "$NEW_CHANGE" -le "$OLD_CHANGE" ]; then
       echo "expected a change with an id greater than $OLD_CHANGE"
+      tests.cleanup restore
```

Comment:
perhaps it is cleaner to call a function that makes these 3 line, so we don't repeat the same in all those places.

echo "$message"
tests.cleanup restore
exit 1